### DISCUSSION TITLE
Empty zips on my windows dev box

### DISCUSSION THREAD

**Discussion by burnacid** [spatie/laravel-personal-data-export#78] [Q&A]
Hi,

First of all thanks for this awesome package.
I have an issue though I have no clue where to look into further.

I'm developing with phpstorm on my windows 11 machine. For some reason the zip files end up empty.
Eventually I've noticed it is creating a folder in C:\Users\******\AppData\Local\Temp\ with the files in it.

So I've pushed a test to my test installation of my project on my linux webserver. And everything ended up working as expected.

Do you have any idea why this isn't properly functioning on Windows?

Laravel 10.7.1
Spatie/laravel-personal-data-export 4.2.1

filesystems.php
```
        'personal-data-exports' => [
            'driver' => 'local',
            'root' => storage_path('app/personal-data-exports'),
        ],
```

Many thanks

**Comment by Rengobli** [comment]
Probably too late but that is a windows thing. Try using 7-Zip and it will work.
